test/skimNtuple_HHbtag.cpp

```diff
@@ -4534,19 +4534,34 @@ int main (int argc, char** argv)
 			    
 		  std::map<std::string, std::vector<float>> result = ParticleNet_SF(tlv_fj.Pt(), PERIOD);
 		  // Set scale factors for different keys
-		  setScaleFactor(result, "HP",
-						 theSmallTree.m_fatjet_particleNetMDJetTags_HP_SF,
-						 theSmallTree.m_fatjet_particleNetMDJetTags_HP_SF_up,
-						 theSmallTree.m_fatjet_particleNetMDJetTags_HP_SF_down);
-		  setScaleFactor(result, "MP",
-						 theSmallTree.m_fatjet_particleNetMDJetTags_MP_SF,
-						 theSmallTree.m_fatjet_particleNetMDJetTags_MP_SF_up,
-						 theSmallTree.m_fatjet_particleNetMDJetTags_MP_SF_down);
-		  setScaleFactor(result, "LP",
-						 theSmallTree.m_fatjet_particleNetMDJetTags_LP_SF,
-						 theSmallTree.m_fatjet_particleNetMDJetTags_LP_SF_up,
-						 theSmallTree.m_fatjet_particleNetMDJetTags_LP_SF_down);
+		  if (isHHsignal){
+		    setScaleFactor(result, "HP",
+				   theSmallTree.m_fatjet_particleNetMDJetTags_HP_SF,
+				   theSmallTree.m_fatjet_particleNetMDJetTags_HP_SF_up,
+				   theSmallTree.m_fatjet_particleNetMDJetTags_HP_SF_down);
+		    setScaleFactor(result, "MP",
+				   theSmallTree.m_fatjet_particleNetMDJetTags_MP_SF,
+				   theSmallTree.m_fatjet_particleNetMDJetTags_MP_SF_up,
+				   theSmallTree.m_fatjet_particleNetMDJetTags_MP_SF_down);
+		    setScaleFactor(result, "LP",
+				   theSmallTree.m_fatjet_particleNetMDJetTags_LP_SF,
+				   theSmallTree.m_fatjet_particleNetMDJetTags_LP_SF_up,
+				   theSmallTree.m_fatjet_particleNetMDJetTags_LP_SF_down);
 
+		  }
+		  else{
+		    theSmallTree.m_fatjet_particleNetMDJetTags_HP_SF = 1.0;
+		    theSmallTree.m_fatjet_particleNetMDJetTags_HP_SF_up = 1.0;
+		    theSmallTree.m_fatjet_particleNetMDJetTags_HP_SF_down = 1.0;
+
+		    theSmallTree.m_fatjet_particleNetMDJetTags_HP_SF = 1.0;
+		    theSmallTree.m_fatjet_particleNetMDJetTags_MP_SF_up = 1.0;
+		    theSmallTree.m_fatjet_particleNetMDJetTags_MP_SF_down = 1.0;
+
+		    theSmallTree.m_fatjet_particleNetMDJetTags_LP_SF = 1.0;
+		    theSmallTree.m_fatjet_particleNetMDJetTags_LP_SF_up = 1.0;
+		    theSmallTree.m_fatjet_particleNetMDJetTags_LP_SF_down = 1.0;
+		  }
```

Comment:
This block is redundant given the defaults. But I understand you might want to explicitly state the we are still missing PNet SFs for the background processes.